consul/default.toml

```diff
@@ -0,0 +1,19 @@
+# If you would like the web gui on the agent
+website = true
+
+# switch this to false you want to start in DEVMODE
+# https://www.consul.io/docs/guides/bootstrapping.html
+[dev]
+mode = true
+
+# The options for consul are available here
+# https://www.consul.io/docs/agent/options.html
+[bootstrap]
+expect = "3"
+
+[server]
+bind = "127.0.0.1"
+data-dir= "/tmp/consul"
```

Comment:
inconsistent spacing

Also, should `/tmp` be the location of the runtime files, or somewhere in the pkg_svc_path?